src/keys.ts

```diff
@@ -448,6 +448,10 @@ function prepareSecretKey(
     return binaryLikeToArrayBuffer(key, encoding);
   }
 
+  if (key instanceof Uint8Array) {
```

Comment:
`binaryLikeToArrayBuffer` here or earlier?

Reply:
Thank you! Done here: https://github.com/margelo/react-native-quick-crypto/pull/320/commits/287ed66c3ab71bfcf55bb6a9bbf5409816a10fe2